src/gtk/toga_gtk/widgets/tree.py

```diff
@@ -80,3 +80,6 @@ def clear(self):
 
     def set_on_select(self, handler):
         pass
+
+    def scroll_to_row(self, row):
```

Comment:
This is a red flag for me - Trees don't have rows, they have nodes. This isn't strictly a problem you've introduced with this patch, but it indicates something we should clean up before we double down on this API.

Reply:
Well, you might say that I did introduce it -- I just moved it from Table to Tree and forgot about the word "row". Good catch!